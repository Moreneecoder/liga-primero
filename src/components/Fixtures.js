import { Link } from 'react-router-dom';
import '../stylesheets/Fixtures.css';
import goalie from '../images/goalie.svg';

const Fixtures = () => (
  <div className="Fixtures px-1 px-md-0 col-12 col-md-6 mx-auto text-center">
    <img className="mb-3" style={{ width: '50%' }} src={goalie} alt="not-found" />
    <div className="alert alert-info p-2">In the Pipes! Coming Soon...</div>
    <Link to="/" className="text-white btn btn-md btn-md-lg main-bg-color">&#8592; Go Back To standings</Link>
  </div>
);

export default Fixtures;
