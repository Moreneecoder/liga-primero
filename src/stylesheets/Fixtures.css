.Fixtures {
  padding-top: 30%;
}

@media only screen and (min-width: 768px) {
  .Fixtures {
    padding-top: 20%;
  }
}

/* @media only screen and (min-width: 1024px) {
  .Fixtures {
    padding-top: 20%;
  }
} */
