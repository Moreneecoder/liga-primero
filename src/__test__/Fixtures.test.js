import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Fixtures from '../components/Fixtures';

describe('Fixtures Component', () => {
  test('Renders Fixtures component as expected', () => {
    const tree = renderer.create(<BrowserRouter>
        <Fixtures />
      </BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
