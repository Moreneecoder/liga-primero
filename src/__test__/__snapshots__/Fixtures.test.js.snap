// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fixtures Component Renders Fixtures component as expected 1`] = `
<div
  className="Fixtures px-1 px-md-0 col-12 col-md-6 mx-auto text-center"
>
  <img
    alt="not-found"
    className="mb-3"
    src="goalie.svg"
    style={
      Object {
        "width": "50%",
      }
    }
  />
  <h3
    className="alert alert-info p-2"
  >
    <span
      className="fw-bolder"
    >
      Fixtures
    </span>
     
    Feature Coming Soon! In the Pipes...
  </h3>
  <a
    className="text-white btn btn-md btn-md-lg main-bg-color"
    href="/"
    onClick={[Function]}
  >
    ‚Üê Go Back To standings
  </a>
</div>
`;
